core/files/configure_misp.sh

```diff
@@ -93,6 +103,33 @@ GPGEOF
     sudo -u www-data /var/www/MISP/app/Console/cake Admin setSetting -q "GnuPG.binary" "$(which gpg)"
 }
 
+set_up_oidc() {
+    if [[ "$OIDC_ENABLE" != "true" ]]; then
+        echo "... OIDC authentication disabled"
+        return
+    fi
+
+    sudo -u www-data php /var/www/MISP/tests/modify_config.php modify "{
```

Comment:
Note that this is not idempotent (can't recall the exact behavior :S).

Best fix, is to reset the value before setting it.

```
sudo -u www-data php /var/www/MISP/tests/modify_config.php modify "{
            \"Security\": {
                \"auth\": \"\"
            }
        }" > /dev/null 
```